internal/tools/go.mod

```diff
@@ -2,6 +2,9 @@ module go.opentelemetry.io/otel/internal/tools
 
 go 1.22.1
 
+// TODO (#3372) remove version replacement when it is compatible. https://github.com/golangci/golangci-lint/issues/3280
+replace github.com/spf13/viper => github.com/spf13/viper v1.12.0
```

Comment:
How about downgrading viper down below rather than a replace? We get golangci-lint breaking changes all the time, and we ignore the renovabot PRs.